src/portal/src/app/system-robot-accounts/new-robot/new-robot.component.ts

```diff
@@ -281,6 +284,17 @@ export class NewRobotComponent implements OnInit, OnDestroy {
       return true;
     }
     if (this.coverAll !== this.coverAllForEdit) {
+      if (this.coverAll) {
```

Comment:
The logic can be sorted out and line 300.

Reply:
Fixed